src/LoRa.cpp

```diff
@@ -582,7 +582,26 @@ void LoRaClass::setOCP(uint8_t mA)
 
 byte LoRaClass::random()
 {
-  return readRegister(REG_RSSI_WIDEBAND);
+  uint8_t currMode = readRegister(REG_OP_MODE);
+  uint8_t retVal = 0;
+
+  while(isTransmitting());
```

Comment:
Does it need a yield() call in this loop so the WDT doesn't fire on platforms like ESP8266/32?